anvio/docs/artifacts/metapangenome.md

```diff
@@ -0,0 +1 @@
+A metapangenome is a way of comparing a bunch of genomes based both on the gene clusters they contain (like in a pangenome) and their abundances in different environments/samples (using metagenomic read recruitment).
```

Comment:
I think we should link to this article here: http://merenlab.org/data/prochlorococcus-metapangenome/ :)

Reply:
Ooooh, that's a good article right there! :)